index.html

```diff
@@ -11,103 +11,104 @@
     <link rel="stylesheet" href="css/style.css" />
     <link rel="stylesheet" href="css/clock.css" />
     <link rel="icon" type="image/x-icon" href="image/favicon.ico" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.0/css/all.min.css"/>
-
+    <link rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.0/css/all.min.css"/>
   </head>
 
   <body>
- <div id="dev-comments-container">
```

Comment:
These are actually required as there is a shortcut to display these. Can you re add an commit